blobstore/cli/clustermgr/config_background.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2023 The CubeFS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package clustermgr
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/desertbit/grumble"
+
+	"github.com/cubefs/cubefs/blobstore/api/clustermgr"
+	"github.com/cubefs/cubefs/blobstore/cli/common"
+	"github.com/cubefs/cubefs/blobstore/cli/common/flags"
+	"github.com/cubefs/cubefs/blobstore/cli/common/fmt"
+	"github.com/cubefs/cubefs/blobstore/common/rpc"
+)
+
+const notSet = "<not set>"
+
+var (
+	BackgroundTaskTypes      = []string{"disk_repair", "balance", "disk_drop", "volume_inspect", "shard_repair", "blob_delete"}
```

Comment:
the task type already defined in `blobstore/common/proto/scheduler.go`,  you can use it

Reply:
Solved. Please have a look at https://github.com/cubefs/cubefs/pull/1997/commits/dca26cb1ef7ad422c4efc768ddd17442c72cc68b .